src/pcapi/core/offers/api.py

```diff
@@ -48,3 +56,26 @@ def list_offers_for_pro_user(
         name_keywords=name_keywords,
         requested_status=requested_status,
     )
+
+
+def post_offer(
+    offer_data: PostOfferBodyModel, current_user: UserSQLEntity
```

Comment:
```suggestion
    offer_data: PostOfferBodyModel, user: UserSQLEntity
```

`current_user` étant aussi une globale fournie par `flask_login`, ça me semble piégeux. (Pas _très_ piégeux, mais un peu. Même si actuellement le seul chemin de code vient de l'API et que ce `user` sera le *current user*, ça ne sera peut-être pas le cas plus tard.)